Give overview of process before details

jameshowison opened the pull request:
Added an overall set of steps to clarify the intended workflow.  The part I was missing was that merging the PR caused a commit on main and thus triggered the build.yaml process that included the push to quay.io

jameshowison committed:
Give overview of process before details

Added an overall set of steps to clarify the intended workflow.  The part I was missing was that merging the PR caused a commit on main and thus triggered the build.yaml process that included the push to quay.io

damianavila reviewed (approved):
LGTM, thanks for adding these clarifications!!

jameshowison committed:
Explain which log shows "pushing to quay.io/..."

Tried to explan which log shows "pushing to quay.io/..." as discussed here: https://github.com/2i2c-org/infrastructure/issues/1158#issuecomment-1089219522

sgibson91 commented on README.md:
```diff
@@ -39,14 +47,16 @@ Step 2 should have provided the appropriate credentials to push the image to qua
 * replace `<quay-username>/<repository-name>` with the info of the `quay.io` repository created at step 2
 * commit the changes you've made to `build.yaml`
 
+If you want to see the log of the Github Action that results you can see this via the Github Actions tab on your image repository. If you are triggering this action by merging a PR you should look at the Github Actions tab (and this will show "pushing quay.io/..."). The build log linked from the PR page is not the one for the merge, but the build done for the PR, prior to the merge (and will ther ...
```
We may want to add something like "... and from the left hand sidselect 'Build container image'   to access logs that push the image, or 'Test container image build' for logs that do not push the image" to this section. Like from the below screenshot.

<img width="724" alt="Screenshot 2022-04-07 at 10 26 56" src="https://user-images.githubusercontent.com/44771837/162167624-e14f5470-77d5-4112-8af5-ae331c6ffac2.png">

sgibson91 reviewed (approved):
LGTM!

GeorgianaElena reviewed (approved):
Thanks @jameshowison! I believe this has great extra info so I'll go ahead and merge this now, then open a follow-up PR to add some more screenshots to where the relevant bits are.

Also, thanks @damianavila and @sgibson91 for the review.🚀
